service/tests/Core.FunctionalTests/Core.FunctionalTests.csproj

```diff
@@ -23,6 +23,7 @@
         </PackageReference>
         <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" />
+        <PackageReference Include="Moq" />
```

Comment:
functional tests should not mock dependencies. If you added some tests with mocks, they should be moved under unit tests

Reply:
You are right, I'll move on the right project